minio/helpers.py

```diff
@@ -406,6 +417,7 @@ def is_valid_policy_type(policy):
 
     return True
 
+
```

Comment:
Formatting changes should be avoided in this PR @maxkulish - please send a different PR for that. Let's keep this close to the required changes.